### PR TITLE
community tab added into the docs.kics.io website

### PR DESCRIPTION
**Pull request by gabriel-cx**
**Proposed Changes**
- Community tab added into the docs.kics.io web page

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: docs/community.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+-   [GitHub discussion regarding Community Meetings](https://github.com/Checkmarx/kics/discussions/5675).
+    -   Here you have access to the PDfs that contains the topics mentioned in each session.
+- Calendar for the upcoming sessions:
+    - October 6, 2022 at 15h00 UTC +0
+    - October 20, 2022 at 15h00 UTC +0
+    - November 3, 2022 at 15h00 UTC +0
+- [To participate in our community meetings you can use this link](https://tinyurl.com/kics-community)
+
+## Hacktoberfest
+*"Hacktoberfest is a month-long celebration of open source projects, their maintainers, and the entire community of contributors. Each October, open source maintainers give new contributors extra attention as they guide developers through their first pull requests on GitHub."*<br />
```
Comment (cxMiguelSilva) [inline docs/community.md:11]:
```suggestion
*"[Hacktoberfest](https://github.com/topics/hacktoberfest) is a month-long celebration of open source projects, their maintainers, and the entire community of contributors. Each October, open source maintainers give new contributors extra attention as they guide developers through their first pull requests on GitHub."*<br />
```

File: docs/community.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<a href="https://github.com/topics/hacktoberfest" style="color: rgb(105, 151, 236);">Hacktoberfest</a>
```
Comment (cxMiguelSilva) [inline docs/community.md:12]:
```suggestion

```

**Review by cxMiguelSilva — CHANGES_REQUESTED** [review]
I think the link to Hacktoberfest should either be incorporated in the sentence or add a quote like:
to know more about Hacktoberfest check the official page

**Review by cxMiguelSilva — APPROVED** [review]
LGTM

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**